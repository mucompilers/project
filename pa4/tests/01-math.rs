fn main() {
      let x = 1 + 1;
      let y = 2 * 2;
      println!("{}", 1 + 2 + 3);
      println!("{}", 1 * 2 % 3);
      println!("{}", x * (1 + 2) / y);
      println!("{}", x * (x + x - x) / x);
      let mut z : i32 = y / x;
      z += 1;
      z *= z + z;
      z /= 2;
      z %= 100;
      println!("{}", z * 3);
}
