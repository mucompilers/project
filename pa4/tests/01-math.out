6
2
1
2
27
